modules/Surveys/Entry/Survey.php

```diff
@@ -378,13 +378,17 @@ function displayClosedPage($survey)
     <div class="container">
         <div class="row">
             <div class="col-md-offset-3 col-md-6">
-                <img height=100 src="modules/Surveys/Entry/survey_logo.jpg"/>
+                <img src="themes/default/images/company_logo.png"/>
```

Comment:
Hi @serhiisamko091184, I imagine this supposed to change if the user changes the system logo. This needs to be dynamically calculated. Plus it should also take into consideration the theme that is being used by the user.

Reply:
Hey @clemente-raposo ,
thanks for feedback, 
done.